pkg/local_object_storage/peapod/read.go

```diff
@@ -0,0 +1,89 @@
+package peapod
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/pkg/util/slice"
+	apistatus "github.com/nspcc-dev/neofs-sdk-go/client/status"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"go.etcd.io/bbolt"
+)
+
+// Get reads data from the underlying database by the given object address.
+// Returns apistatus.ErrObjectNotFound if object is missing in the Peapod.
+func (x *Peapod) Get(addr oid.Address) ([]byte, error) {
+	var data []byte
+
+	err := x.bolt.View(func(tx *bbolt.Tx) error {
+		bktRoot := tx.Bucket(rootBucket)
+		if bktRoot == nil {
+			return fmt.Errorf("%w: missing root bucket", apistatus.ErrObjectNotFound)
```

Comment:
looks like false positive `errors.Is(err, apistatus.ErrObjectNotFound)` on the caller side then while missing that bucket is more like a bug according to the current `New`

Reply:
> but the fact remains - the object was not found

sure, but potential problems scare me: we even are impossible to find a potential bug: read operations are OK, write operations are redirected to another storage

Reply:
if bucket is missing for any reason, both read and writes ops will fail, so i didn't got ur concern point. Error message contains detailed reason

Reply:
Likely you can panic here right away. Or declare this shard as dead one. Root bucket must be initialized along with the DB, if it's missing we're in big trouble.

Reply:
yeah, root bucket is vital one, my mind is still on root-cnr-obj reality. So both of u right. I guess panic isn't an option since this ain't a programmer mistake, so i'll return some error which will be interpreted as common non-status runtime error